Assets/Features/EventManager/EventId.cs

```diff
@@ -2,6 +2,8 @@
 {
     public enum EventId
```

Comment:
Events are most often a trigger that something has happened. Changing the names to make them start with 'on' might clarify this. I.e. onBrushChanged, onBuildingUnlocked etc.

Reply:
Although... This is an enum, so this might not be applicable to this. In that case, it's just a future suggestion or something to keep in mind